google-chart.html

```diff
@@ -197,6 +199,25 @@
           this.options = {};
           this.rows = [];
           this.dataTable = null;
+
+          var head = document.querySelector('head');
+          var element = this;
+          var observer = new MutationObserver(function(mutations) {
+              mutations.forEach(function(mutation) {
+                for (var i = 0; i < mutation.addedNodes.length; i++) {
+                  var newNode = mutation.addedNodes[i];
+                  if (newNode.rel && newNode.rel == 'stylesheet') {
+                    //Add the stylesheet as an @import
+                    var cssImport = document.createElement('style');
+                    cssImport.innerHTML = "@import '" + newNode.href + "';";
+                    element.shadowRoot.appendChild(cssImport);
```

Comment:
Another option would be to work the the gviz team to have them expose the stylesheet URLs associated with each individual chart type. This is probably the best long-term strategy, as it guarantees we'd always been in sync.

There could also be a load configuration option that tells gviz libraries not to load their own styles, and to let consumer take care of it. This would prevent unnecessary styles from existing on the page.